packages/service-clients/end-to-end-tests/odsp-client/src/test/audience.spec.ts

```diff
@@ -24,15 +24,11 @@ describe("Fluid audience", () => {
 	const connectTimeoutMs = 10_000;
 	let client: OdspClient;
 	let schema: ContainerSchema;
-	const client1Creds: IOdspLoginCredentials = {
-		username: process.env.odsp__client__login__username as string,
-		password: process.env.odsp__client__login__password as string,
-	};
+	const [client1Creds, client2Creds] = getCredentials();
 
-	const client2Creds: IOdspLoginCredentials = {
-		username: process.env.odsp__client2__login__username as string,
-		password: process.env.odsp__client2__login__password as string,
-	};
+	if (client1Creds === undefined || client2Creds === undefined) {
+		throw new Error("Invalid login credentials");
```

Comment:
Nit, might be slightly clearer description of the error (here and other tests too).
```suggestion
		throw new Error("Couldn't get login credentials");
```